src/runner.rs

```diff
@@ -191,13 +194,16 @@ pub mod platform {
     use winreg::enums::*;
 
     pub fn firefox_default_path() -> Option<PathBuf> {
+        if let Some(path) = find_binary("firefox.exe") {
+            return Some(path)
+        }
         let opt_path = firefox_registry_path().unwrap_or(None);
```

Comment:
The behaviour is already inconsistent between platforms; Windows has a registry and OSX/linux don't. I think that if we wanted an environment variable to mean "use the firefox in this location unless there's a capability set" we should invent `$GECKODRIVER_BINARY` or similar.

For now let's revert the Windows part of the change.